packages/client/hmi-client/src/components/workflow/ops/model-comparison/tera-model-comparison.vue

```diff
@@ -191,13 +194,21 @@ const sampleAgentQuestions = [
 
 const isLoadingStructuralComparisons = ref(false);
 const structuralComparisons = ref<string[]>([]);
-const llmAnswer = ref('');
+const compareModelsTaskId = ref<string>('');
+const compareModelsTaskOutput = ref<string>('');
 const code = ref(props.node.state.notebookHistory?.[0]?.code ?? '');
 const llmThoughts = ref<any[]>([]);
 const isKernelReady = ref(false);
 const modelsToCompare = ref<Model[]>([]);
 const contextLanguage = ref<string>('python3');
 
+const llmAnswer = computed(() => {
+	if (!compareModelsTaskOutput.value) return '';
+	const str = b64DecodeUnicode(compareModelsTaskOutput.value);
+	const parsedValue = JSON.parse(str) as CompareModelsResponseType;
+	return parsedValue.response;
```

Comment:
Why is this not done in line 338 when we read the `taskRed.output`?

Reply:
We assign the output in line 345 (and also 338) as well. So instead of duplicating parsing logics in two places, we just do once here.

Reply:
Yes, but in a computed seems a bit much no?